pages/search.js

```diff
@@ -1,11 +1,12 @@
-import { useEffect, useState, useRef } from "react";
+import { useEffect, useState, useRef, useCallback } from "react";
 import { useRouter } from "next/router";
 
 import UserCard from "../components/user/UserCard";
 import Alert from "../components/Alert";
 import Page from "../components/Page";
 import PageHead from "../components/PageHead";
 import Tag from "../components/Tag";
+import { debounce } from "lodash";
```

Comment:
Is it possible to do it without `lodash`? Maybe with a `setTimeout`

Reply:
Thanks for the suggestion eddie it work better with `setTimeout` learned one more thing today :)
also I attacked a screen recording for the same to

Reply:
Great 👍